demo/text-classification/predict.py

```diff
@@ -33,107 +33,35 @@
-        metrics_choices = ["acc"]
-    elif args.dataset.lower() == 'thucnews':
-        dataset = hub.dataset.THUCNEWS()
-        module = hub.Module(name="roberta_wwm_ext_chinese_L-24_H-1024_A-16")
-        metrics_choices = ["acc"]
-    elif args.dataset.lower() == 'iflytek':
-        dataset = hub.dataset.IFLYTEK()
-        module = hub.Module(name="roberta_wwm_ext_chinese_L-24_H-1024_A-16")
-        metrics_choices = ["acc"]
-    elif args.dataset.lower() == "mrpc":
-        dataset = hub.dataset.GLUE("MRPC")
-        module = hub.Module(name="ernie_v2_eng_base")
-        metrics_choices = ["f1", "acc"]
-    # The first metric will be choose to eval. Ref: task.py:799
-    elif args.dataset.lower() == "qqp":
-        dataset = hub.dataset.GLUE("QQP")
-        module = hub.Module(name="ernie_v2_eng_base")
-        metrics_choices = ["f1", "acc"]
-    elif args.dataset.lower() == "sst-2":
-        dataset = hub.dataset.GLUE("SST-2")
-        module = hub.Module(name="ernie_v2_eng_base")
-        metrics_choices = ["acc"]
-    elif args.dataset.lower() == "cola":
-        dataset = hub.dataset.GLUE("CoLA")
-        module = hub.Module(name="ernie_v2_eng_base")
-        metrics_choices = ["matthews", "acc"]
-    elif args.dataset.lower() == "qnli":
-        dataset = hub.dataset.GLUE("QNLI")
-        module = hub.Module(name="ernie_v2_eng_base")
-        metrics_choices = ["acc"]
-    elif args.dataset.lower() == "rte":
-        dataset = hub.dataset.GLUE("RTE")
-        module = hub.Module(name="ernie_v2_eng_base")
-        metrics_choices = ["acc"]
-    elif args.dataset.lower() == "mnli" or args.dataset.lower() == "mnli_m":
-        dataset = hub.dataset.GLUE("MNLI_m")
-        module = hub.Module(name="ernie_v2_eng_base")
-        metrics_choices = ["acc"]
-    elif args.dataset.lower() == "mnli_mm":
-        dataset = hub.dataset.GLUE("MNLI_mm")
-        module = hub.Module(name="ernie_v2_eng_base")
-        metrics_choices = ["acc"]
-    elif args.dataset.lower().startswith("xnli"):
-        dataset = hub.dataset.XNLI(language=args.dataset.lower()[-2:])
-        module = hub.Module(name="roberta_wwm_ext_chinese_L-24_H-1024_A-16")
-        metrics_choices = ["acc"]
-    else:
-        raise ValueError("%s dataset is not defined" % args.dataset)
-
-    support_metrics = ["acc", "f1", "matthews"]
-    for metric in metrics_choices:
-        if metric not in support_metrics:
-            raise ValueError("\"%s\" metric is not defined" % metric)
-
+    # Load Paddlehub ERNIE Tiny pretrained model
+    module = hub.Module(name="ernie_tiny")
     inputs, outputs, program = module.context(
         trainable=True, max_seq_len=args.max_seq_len)
+
+    # Download dataset and use accuracy as metrics
+    # Choose dataset: GLUE/XNLI/ChinesesGLUE/NLPCC-DBQA/LCQMC
+    dataset = hub.dataset.ChnSentiCorp()
+
+    # For ernie_tiny, it use sub-word to tokenize chinese sentence
+    # If not ernie tiny, sp_model_path and word_dict_path should be set None
```

Comment:
If not ernie tiny, module.get_spm_path() and module.get_word_dict_path() will return None